packages/plugin-node-in-project/in-project.js

```diff
@@ -0,0 +1,16 @@
+const { map } = require('@bugsnag/core/lib/es-utils')
+const normalizePath = require('@bugsnag/core/lib/path-normalizer')
+
+module.exports = {
+  init: client => client.config.beforeSend.push(report => {
+    report.stacktrace = map(report.stacktrace, stackframe => {
+      const projectRoot = normalizePath(client.config.projectRoot)
```

Comment:
Shouldn't this be outside of the loop since it won't change per frame?

Reply:
Yes, good spot, thanks!